### ISSUE TITLE
Feature: User > The menu and the selection dishes

### ISSUE THREAD

**Issue by MikaelFeher** [CraftAcademy/slow_food_june#5]
As a user
In order to be able to select a dish I like
I would like to see a menu of dishes

**Comment by tochman** [comment]
If I may, I would rephrase this: 

```
As a user 
In order to be able to select a dish I like
I would like to see a menu of dishes
```

Now, think about the business objective `to select a dish I like`, imagine you are the **stakeholder** and ask yourself the question **`Why is that important?`**. Once you have answered that question, take the answer and ask the same question again. Repeat that process 5 - 6 times. What do you end up with?  

My theory is that at some point you reach a point where the answer is "So I can **MAKE** more money" or "So I can **AVOID LOOSING/SPENDING** money". It can also be a non-monetary gain for some stakeholders but generally speaking a good user story's business objective should be mapped to $$$. ;-)

Make a habit of doing this for all user stories in this little project. It's a good exercise.

**Comment by rdcsg** [comment]
`In order to be able to select a dish I like`
- because I like to find the things I like in a menu

**Comment by rdcsg** [comment]
`In order to be able to select a dish I like``
- because I can remember for later that I can find it here

**Comment by rdcsg** [comment]
`In order to be able to select a dish I like``
- because then I can come back and get it again

**Comment by rdcsg** [comment]
`In order to be able to select a dish I like``
- and if it is real good and the service and the rest of it is good, probably I will come back regularly, or at least more than once every now and then  =)

**Comment by rdcsg** [comment]
`In order to be able to select a dish I like`
- so I really want to see what goodies n stuff are in here =)

**Comment by rdcsg** [comment]
**Feature:** As a user
In order to be able to select and buy the tasty dishes I like
I need to see a menu with the dishes 

**Scenario:** Allows a user to select dishes
_Given_ I have a list of categories of dishes and descriptions of each dish (content and price)
_And_ I can select the dishes I like
_Then_ They should be added to my shopping bag / cart

**Comment by tochman** [comment]
TODO:

We need to display some dishes on a separate page
1. create a route to dishes show page ( GET `/dishes`)
2. create a template (`views/dishes.erb`)
3. create a `Dish` model class (extend wit DataMapper functionality) 
4. grab all `@dishes` from the database (in the controller) 
   4 create controller action to pass in the dishes to the template (together wit the route) 
5. display the `@dishes` on the template
